--- a/utils/clone.py
+++ b/utils/clone.py
@@ -172,17 +172,18 @@
     # Follow tarball procedure:
     # https://source.chromium.org/chromium/chromium/tools/build/+/main:recipes/recipes/publish_tarball.py
-    get_logger().info('Downloading pgo profiles')
-    run([
-        sys.executable,
-        str(args.output / 'tools' / 'update_pgo_profiles.py'), '--target=' + args.pgo, 'update',
-        '--gs-url-base=chromium-optimization-profiles/pgo_profiles'
-    ],
-        check=True)
-    # https://chromium-review.googlesource.com/c/chromium/tools/build/+/4380399
-    run([
-        sys.executable,
-        str(args.output / 'v8' / 'tools' / 'builtins-pgo' / 'download_profiles.py'), 'download',
-        '--depot-tools',
-        str(dtpath)
-    ],
-        check=True)
+    if not environ.get('HELIUM_SKIP_PGO'):
+        get_logger().info('Downloading pgo profiles')
+        run([
+            sys.executable,
+            str(args.output / 'tools' / 'update_pgo_profiles.py'), '--target=' + args.pgo, 'update',
+            '--gs-url-base=chromium-optimization-profiles/pgo_profiles'
+        ],
+            check=True)
+        # https://chromium-review.googlesource.com/c/chromium/tools/build/+/4380399
+        run([
+            sys.executable,
+            str(args.output / 'v8' / 'tools' / 'builtins-pgo' / 'download_profiles.py'), 'download',
+            '--depot-tools',
+            str(dtpath)
+        ],
+            check=True)

    get_logger().info('Generating: DAWN_VERSION')